pkg/api/customization/workload/workload.go

```diff
@@ -144,6 +157,30 @@ func (a ActionWrapper) rollbackDeployment(apiContext *types.APIContext, clusterC
 	if revision == "" {
 		return httperror.NewAPIError(httperror.ServerError, fmt.Sprintf("ReplicaSet %s doesn't exist for deployment %s", rollbackInput.ReplicaSetID, deployment.ID))
 	}
+	deploymentVersion, err := k8sschema.ParseGroupVersion(depl.APIVersion)
```

Comment:
we don't need to call L155 if we're already getting replicaSet ourselves, maybe this needs to be moved up a bit